lutris/installer/commands.py

```diff
@@ -173,16 +173,15 @@ def _on_input_menu_validated(self, widget, *args):
     def insert_disc(self, data):
         self._check_required_params('requires', data, 'insert_disc')
         requires = data.get('requires')
-        message = data.get(
-            'message',
-            "Insert or mount game disc and click Autodetect or\n"
-            "use Browse if the disc is mounted on a non standard location."
-        )
-        message += (
-            "\n\nLutris is looking for a mounted disk drive or image \n"
-            "containing the following file or folder:\n"
-            "<i>%s</i>" % requires
-        )
+        message_action = _("Insert or mount game disc and click Autodetect or") + "\n"
```

Comment:
This shouldn't be split like this, it turns a normal paragraph into weird partial sentences.